hyperspy/_signals/signal1d.py

```diff
@@ -1046,6 +1056,7 @@ def remove_background(
                 signal_range=signal_range,
                 background_estimator=background_estimator,
                 fast=fast,
+                zero_fill=zero_fill,
```

Comment:
At line 1038: similarly to here, I would also add the possibility to pass the "zero_fill" (also the "background_estimator" and the "fast", while you are at it!), in order to be able to preset the option from the notebook or the qtconsole.

Reply:
I'm not sure I understand what you mean. Do you mean in the call to the `BackgroundRemoval` constructor?

Reply:
Yes! The idea is to be able to do, for example:
```python
s.remove_background(fast=False, zero_fill=True)
```
and get the option accordingly set in the UI.